### PR TITLE
set mask to false for nc-region. 

### PR DESCRIPTION
**Pull request by huard**
Otherwise error is raised about missing mask when writing to disk. This is mysterious to me...

### CODE REVIEW

**Comment by bekozi** [comment]
Thanks, @huard. I wonder if this is a bug somewhere in the mask handling. How do I reproduce the "missing mask when writing to disk" error?

**Comment by huard** [comment]
Test file sent by email.

File: src/ocgis/conv/nc.py [hunk 1]
```diff
@@ -169,14 +169,13 @@ def _preformatting_(self, i, coll):
                                )
             )
 
-            if 'ocgis_spatial_mask' in field:
-                # Remove the spatial_mask and replace by new one.
-                field.remove_variable('ocgis_spatial_mask')
+            if VariableName.SPATIAL_MASK in field:
+                # Remove the spatial_mask.
+                field.remove_variable(VariableName.SPATIAL_MASK)
+                field.dimension_map.set_spatial_mask(None)
```
Comment (bekozi) [inline src/ocgis/conv/nc.py]: @huard: The spatial mask variable retrieval is strict when it is listed in the dimension map. This should be relaxed (going to think about that), but in the meantime this is fully correct. If these changes work for you, I'll merge!

**Comment by tomLandry** [comment]
Great work guys!

**Comment by huard** [comment]
Good for me. Please go ahead.

**Comment by bekozi** [comment]
Okay! Thanks for bringing up this disconnect. Keeping metadata and operations in sync is often challenging.

Also thanks @tomLandry for the props. Hope all is well!

**Pull request merged**